spec/index.html

```diff
@@ -1498,6 +1498,51 @@ <h2>RDF Datasets</h2>
     between datasets would require RDF graph entailments between the graphs with the same name
     (adding in empty graphs as required).</p>
 
+  <table>
+    <caption>Definition of an RDF dataset</caption>
+    <tr>
+       <td class="semantictable">
+       <p>An <dfn data-lt="RDF dataset">RDF dataset</dfn> I is a set:</p>
```

Comment:
RDF Dataset is actually defined in RDF Concepts, although it has an alias here, so it's sufficient to just use an anchor.
```suggestion
       <p>An <a>RDF dataset</a> I is a set:</p>
```suggestion

Reply:
The dfn markup is copied from examples elsewhere.  If they are wrong here then I think there are lots that have to be changed throughout the document set.